pom.xml

```diff
@@ -164,6 +164,11 @@
                 <artifactId>worker-jobtracking-shared</artifactId>
                 <version>3.5.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.stefanbirkner</groupId>
```

Comment:
@dermot-hardy I wasn't sure if this should go into the https://github.com/CAFapi/caf-dependency-management-bom/blob/master/pom.xml or not - should all new dependencies go in there?

Reply:
What you've done is fine.  I think we'll need a rethink on `caf-dependency-management-bom`.